client/landing/stepper/declarative-flow/internals/steps-repository/import-ready-not/index.tsx

```diff
@@ -2,10 +2,16 @@ import React from 'react';
 import { ReadyNotStep } from 'calypso/blocks/import/ready';
 import { Step } from 'calypso/landing/stepper/declarative-flow/internals/types';
 import { recordTracksEvent } from 'calypso/lib/analytics/tracks';
+import { ImporterPlatform } from 'calypso/lib/importer/types';
```

Comment:
Nit: another type import
```suggestion
import type { ImporterPlatform } from 'calypso/lib/importer/types';
```